flake.nix

```
@@ -190,6 +190,8 @@
           };
         };
 
+        formatter = pkgs.nixfmt-tree.override { settings.width = 80; };
```

Consider also overriding `settings.tree-root-file`. By default, `nixfmt-tree` sets it to `.git/index`, however this assumes a) a git repo and b) not using git worktrees.

If you have some filepath you know will always be present relative to the root of your repo, you should ideally set `settings.tree-root-file` to it.

Looking at your repo, `flake.lock`, `flake.nix`, or `LICENSE` could work.

```suggestion
        formatter = pkgs.nixfmt-tree.override {
          settings.tree-root-file = "flake.lock";
          settings.width = 80;
         };
```